package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "neverthrow",
-  "version": "4.3.1",
+  "version": "4.4.2",
```

Comment:
Something seems odd here ... the `master` branch is already at 4.4.2, so we shouldn't be seeing a diff here.